src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -492,6 +492,24 @@ public Invoice getInvoiceByIdOrNumber(final String invoiceIdOrNumber, final bool
 
         return httpClient.doGet(uri, queryParams, Invoice.class);
     }
+    
+    public String getInvoiceAsHtml(final String invoiceId) throws KillBillClientException {
+      final String uri = JaxrsResource.INVOICES_PATH + "/" + invoiceId + "/" + JaxrsResource.INVOICE_HTML;
+      final Multimap<String, String> queryParams = ImmutableMultimap.<String, String>of(KillBillHttpClient.HTTP_HEADER_ACCEPT, KillBillHttpClient.ACCEPT_HTML);        
+      
+      final Response response = httpClient.doGet(uri, queryParams, Response.class);
+      
+      if(response != null) {
+        try {
+          return response.getResponseBody();
+        }
+        catch (final IOException e) {
+          throw new KillBillClientException(e);
+        }        
+      } else {
+        return null;
+      }
+    }
```

Comment:
Could you maybe reuse the [getResourceFile](https://github.com/killbill/killbill-client-java/blob/master/src/main/java/org/killbill/billing/client/KillBillClient.java#L1547) instead of re-writing it differently?